JenkinsfileRT

```diff
@@ -64,7 +64,8 @@ bc0.build_cmds = [
 bc0.build_cmds = PipInject(env.OVERRIDE_REQUIREMENTS) + bc0.build_cmds
 bc0.test_cmds = [
     "pytest --cov-report=xml --cov -r sxf -n 30 --bigdata --slow \
-    --basetemp=${pytest_basetemp} --junit-xml=results.xml --dist=loadscope",
+    --basetemp=${pytest_basetemp} --junit-xml=results.xml --dist=loadscope \
+    --env=${env.ARTIFACTORY_ENV} ",
```

Comment:
When this gets merged, we need up update all the build jobs (`JWST`, `JWST_UNSTABLE`, `jdavies-dev`, and `eslavich-dev`) that run the regression tests with this parameter.

Reply:
would things horribly crash without the jobs being updated. just curious.

Reply:
Yes.  Especially since there's no definition of this anywhere in the JenkinsfileRT script other than here.  Notice how `env.OVERRIDE_REQUIREMENTS` is sorted out at the top of this script.

Reply:
I can add similar code, defaulting to `dev`.